src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -147,7 +147,8 @@ protected override void InitEditor(bool fresh)
 
         reportTab.UpdateReportTabPatchSelector();
 
-        reportTab.UpdateGlucoseReduction(CurrentGame.GameWorld.WorldSettings.GlucoseDecay);
+        reportTab.UpdateEvents(CurrentGame.GameWorld.EventsLog.Where(
+            c => c.Key == CurrentGame.GameWorld.TotalPassedTime).ToDictionary().Values.ToList()[0]);
```

Comment:
We'll probably have events in the future that last multiple generations so I think the events log should be an object that can be asked here if there are any ongoing events at current editor time.

Reply:
I tried making system show eruption marker on a patch map node if there is event in events log but it didn't work. I think we can ignore possible multi-generations events for now and handle them when we actually implement these